include/geode/geometry/point.hpp

```diff
@@ -174,6 +174,27 @@ namespace geode
             return oss.str();
         }
 
+        [[nodiscard]] Point< dimension - 1 > project_point(
+            geode::local_index_t axis_to_remove ) const
+        {
+            OPENGEODE_ASSERT( axis_to_remove < dimension && axis_to_remove >= 0
+                                  && dimension > 1,
+                "[Point] Invalid axis to remove" );
+            OPENGEODE_ASSERT(
+                dimension > 1, "[Point] Invalid dimension to reduce" );
+            Point< dimension - 1 > projected_point;
+            geode::index_t dim{ 0 };
+            for( const auto i : LRange{ dimension } )
+            {
+                if( i != axis_to_remove )
+                {
+                    projected_point.set_value( dim, this->value( i ) );
```

Comment:
By doing so you may return primitives. We should choose which algorithm to do if we want to remove coord 1 : either 0,2 or 2,0 !

Reply:
We already had this kind of issues in rasterize functions

Reply:
I would say 0, 2 to keep the same order, we could implement a reorder/permute method if needed

Reply:
How do you want to choose an order if not keeping the same ?